repo/packages/S/spark/0/mantl.json

```diff
@@ -0,0 +1,5 @@
+{
+  "mantl": {
+    "load-balancer": "external"
```

Comment:
We've been configuring most of the packages to not expose themselves via traefik by default. It might not be desired to expose the UI to the public without authentication. Also, I notice the UI doesn't quite work anyway (it tries to redirect to internal dns names (*.service.consul)) that don't work when you are accessing it from outside the cluster. Can we default this to "off"?

Once we have this package working, we might be able to add the spark ui into mantl UI so that it just shows up when spark is running on the cluster.